test/lint/lint-python.sh

```diff
@@ -39,7 +39,6 @@ enabled=(
     E711 # comparison to None should be 'if cond is None:'
     E714 # test for object identity should be "is not"
     E721 # do not compare types, use "isinstance()"
-    E741 # do not use variables named "l", "O", or "I"
```

Comment:
Going to add this back in a followup PR with fixes to offending code.

Reply:
Perhaps it can be done in this PR?

Reply:
I guess that would make it more atomic :)

There doesn't seem to be too much going on in this PR so I am happy to fix these here too if wanted. 👍

Reply:
Renaming variables to comply to E741, imo, deserves its own PR. It would be nice if it will a scripted-diff.

Reply:
After checking, the usage is also not so uniform throughout. `l` is in place of `listener`, `line`, etc and there are occurrences of it as a function parameter. Seems there might be other further discussion on that too.